docs/resources/recognition.md

```diff
@@ -0,0 +1,46 @@
+# Recognizing Contributions
+
+There are a variety of ways that we would like to recognize community contributions to hackweeks. 
+
+## Website Presence
+
+We invite all people who join us in organizing and hosting a hackweek to be included in our "Meet the team" section of each hackweek website. 
+
+We use this YAML markdown template for each team member. 
+
+```
+- title: 
+  avatar: 
+  role: 
+  organizations:
+  - name: 
+    url: 
+  - name: 
+  bio: 
+  expertise:
+  - 
+  - 
+  social:
+  - icon: github
+    icon_pack: fab
+    link: 
+  user_groups:
+  - 
+  ```
+
+The first step is to copy the text above into your favorite editor and add your information. Here is an example of how the text you include gets rendered on the website:
+
+  ```{image} ../images/website-bio.jpg
+:alt: example of how to fill out website bio
+:width: 1100px
+:align: center
+```
+
+Note that it is important to precisely follow the indentation and spacing layout shown here, so that the website correctly interprets the YAML format. Also, we encourage everyone to use their GitHub avatar as the source of their personal photograph.

```

Comment:
```suggestion
**Notes:**
- You must precisely follow the indentation and spacing layout shown here, so that the website correctly interprets the YAML format.
- We encourage everyone to use their GitHub avatar as the source of their personal photograph.
- Only your GitHub username (not a full url) is required for the `link` key under `social`.
- Acceptable user groups (roles) can be found [here](https://uwhackweek.github.io/hackweeks-as-a-service/services/index.html).

```